front/components/assistant_builder/NamingScreen.tsx

```diff
@@ -168,6 +173,39 @@ export default function NamingScreen({
     }
   }, [owner, builderState.instructions, builderState.description]);
 
+  const updateEmojiFromSuggestions = useCallback(async () => {
```

Comment:
So this only starts loading once we reach the naming screen ?

Reply:
Yes!